pkg/boot/linux.go

```diff
@@ -174,6 +175,33 @@ func (li *LinuxImage) String() string {
 	)
 }
 
+func isTmpfsReadOnlyFile(f *os.File) bool {
+	if f == nil {
+		return false
+	}
+
+	log.Printf("Start checking for file: %v", f)
```

Comment:
probably need to remove all these logs from pkg functions?

Reply:
Done!